### PR TITLE
Maintenantce for service not found

### PR DESCRIPTION
**Pull request by sergeysenja1992**

### CODE REVIEW

File: packages/dynamic/src/loader/xm-dynamic-module-registry.service.ts [hunk 1]
```diff
@@ -19,7 +19,7 @@ export class XmDynamicModuleRegistry {
         const flattened = flatten(flatten(dynamicExtensions));
 
         const duplicates = filter(flattened, (o, i) =>
-            find(flattened, (e) => e.selector == o.selector, i + 1)
+            find(flattened, (e) => e.selector == o.selector && e.selector !== 'xm-general', i + 1)
```
Comment (Kaskyi) [inline packages/dynamic/src/loader/xm-dynamic-module-registry.service.ts:22]: nonono

File: packages/cli/src/ext-lazy-module-command.ts [hunk 2]
```diff
@@ -36,20 +38,49 @@ export class ExtLazyModuleCommand implements Command {
...
+            tsConfigFilePath: 'tsconfig.json',
+        });
+
+        const sourceFiles = project.getSourceFiles();
+        for (const sourceFile of sourceFiles) {
+            const components = sourceFile.getClasses()
+                .filter(i => !!i.getDecorator('Component'));
+            for (const component of components) {
+                const selector = getSelectorValue(component);
+                if (selector?.startsWith('xm-general/')) {
```
[the remark this answers is not in the document]
Comment (sergeysenja1992) [reply packages/cli/src/ext-lazy-module-command.ts:52]: yes yes yes ))

File: packages/dynamic/src/loader/xm-dynamic-module-registry.service.ts (hunk 1 above)
Comment (sergeysenja1992) [reply packages/dynamic/src/loader/xm-dynamic-module-registry.service.ts]: updated

**Review by mak7iliano — APPROVED** [review]

**Comment by sergeysenja1992** [comment]
Discussed that way will be updated after meeting with @Kaskyi.

**Pull request merged**